ro_help/hub/templatetags/custom_tags.py

```diff
@@ -0,0 +1,20 @@
+import os

```

Comment:
`custom_tags` is a little too generic and all the tags that are defined by us are custom.
Maybe we can name it `meta_tags`, following the pattern from https://github.com/code4romania/ro-help/pull/144/files

Reply:
I put it like this because i wasn't sure if we would use one file for all custom tags or one file per tag. I have renamed the file accordingly.